src/AuthenticatedAPIClient/axiosConfig.js

```diff
@@ -94,13 +95,20 @@ function applyAxiosInterceptors(authenticatedAPIClient) {
     });
   }
 
-  // Redirect to the logout page if an unauthorized API response was received.
+  // Log errors and info for unauthorized API responses
   function handleUnauthorizedAPIResponse(error) {
-    const errorStatus = error && error.response && error.response.status;
-    if (errorStatus === 401 || errorStatus === 403) {
+    const response = error && error.response;
+    const errorStatus = response && response.status;
+    const requestUrl = response && response.config && response.config.url;
+    const requestIsTokenRefresh = requestUrl === authenticatedAPIClient.refreshAccessTokenEndpoint;
+
+    if (errorStatus === 401 && !requestIsTokenRefresh) {
       logAPIErrorResponse(error, { errorFunctionName: 'handleUnauthorizedAPIResponse' });
-      authenticatedAPIClient.logout(authenticatedAPIClient.appBaseUrl);
     }
+    if (errorStatus === 403) {
+      logInfo(`Forbidden API response from ${requestUrl}`);
```

Comment:
1. Again, not sure if we want to log if we aren't handling this.
2. The old code used to have default handling for `(errorStatus === 401 && !requestIsTokenRefresh) || errorStatus === 403)` that would log error and logout.  Seems like we might still want that here, unless someone passes a flag or separate handler from the one above.  A handler could be named something like `handleAPIAuthAndAuthzErrors`, but not sure how it would interact with the handler.  So, a flag like existing in @adamstankiewicz's original PR might be more appropriate in this circumstance?
Note: In all cases, we should make it clear that this code is now for responses from API calls in general, and **not** the LoginRefresh/Authentication code.

Reply:
About #2 - We queue a token refresh request before any request if we have an expired access token. I think this means we shouldn’t ever be in a situation where we get a 401 for any request that’s not the login refresh. If we do want to handle that case we should delete the catch above for our login refresh request and handle all of it here instead.

Reply:
Or mayb add a special case to ensure what we think is happening is true, and log an error if it isn't?